### PR TITLE
enhance navbar

### LINKED ISSUE #40: Enhance navbar
Enhance the navbar and add links. I am working on this issue

### PR DESCRIPTION
**Pull request by pythongiant**
Add necessary links to navbar. Add smooth scroll.
fixed #40 
https://pythongiant.github.io/2019.fossasia.org

### CODE REVIEW

**Comment by pythongiant** [comment]
CC: @realslimshanky

**Comment by pythongiant** [comment]
@jogendra done!

File: index.html [hunk 1]
```diff
@@ -152,10 +151,21 @@
                     </li>
                     <li>
                         <a target="_self" class="inner-link" href="#press">Press</a>
+                    </li>-->
+                    <li>
+                        <a target="_self" href="#topics">Topics</a>
+                    </li>
+                    
+                    <li>
+                        <a target="_self" href="#about">About</a>
```
Comment (jogendra) [inline index.html]: This is unnecessary.

Comment (pythongiant) [reply index.html]: what is? The white space?

Comment (pythongiant) [reply index.html]: or the about link?

**Comment by pythongiant** [comment]
@mariobehling

File: index.html (hunk 1 above)
Comment (jogendra) [reply index.html]: About link, it is unnecessary for navbar

**Comment by pythongiant** [comment]
@sahilsaha7773 thats due to the fact that we dont have a mobile navbar yet. #35 solved this issue.

**Comment by pythongiant** [comment]
CC : @abishekvashok

**Pull request closed without merging**